dockerfiles/centos-build/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM centos/devtoolset-7-toolchain-centos7
```

Comment:
This was a way to build easily locally on any machine w/ docker installed.
This should also work on azure-pipelines build agents but I did not hook that up here.

Reply:
Hi Ben,

Thank you for the PR. Did you test how will consumer of the package in .net know which DLL to load or where to load it from? 

Thanks!
-Sergei

Reply:
Also could you please sign the required CLA above?

Reply:
Hi Sergei
I'm waiting to receive approval to sign the CLA hopefully I can to that soon.
The dll from the centos.7 folder gets published when this command is run:
`dotnet publish -r centos.7-x64`
If you publish for linux-x64 you get the generic version.